storage/s3.go

```diff
@@ -799,9 +799,12 @@ func (sc *SessionCache) newSession(ctx context.Context, opts Options) (*session.
 		WithEndpoint(endpointURL.String()).
 		WithS3ForcePathStyle(!isVirtualHostStyle).
 		WithS3UseAccelerate(useAccelerate).
-		WithHTTPClient(httpClient).
-		WithLogLevel(aws.LogDebug).
-		WithLogger(sdkLogger{})
+		WithHTTPClient(httpClient)
+
+	if opts.LogLevel == "trace" {
```

Comment:
We have log levels in `log` package. It should be safe to export them and use it here.

Reply:
I've exported the LogLevel types and changed this line to
`log.LevelFromString(opts.LogLevel) != log.LevelTrace `.

I've 2 questions:
1) Should I add a bullet point saying "LogLevel type is exported..." to Changelog?

2) Should type of storage.Options.LogLevel remain as string or should I change it to LogLevel too.
    > ps. all other options are set with c.String, c.Bool and c.Int methods in  [NewStorageOpts](https://github.com/peak/s5cmd/blob/d6f9d7af625ca1dd8f0941704d09e1a744c136fc/command/app.go#L156-#L167).
If we change its type we this would be set with `log.LevelFromString(c.String("log"))`, different than other option parameters.

Reply:
1. no need as s5cmd doesn't provide an API.
2. it'd better to use the type.